docs/pages/machine-id/access-guides/applications.mdx

```diff
@@ -5,200 +5,106 @@ description: How to use Machine ID to access Applications
 
 (!docs/pages/includes/machine-id/v2-config-warning.mdx!)
 
-This guide demonstrates how to configure an automated service to use
-Machine ID to access a web application protected by Teleport.
+Teleport Application Access protects and controls access to HTTP and TCP
+applications. Machine ID can be used to grant machines secure, short-lived
+access to these applications.
 
-With Machine ID, Teleport issues short-lived certificates, tied to a machine
```

Comment:
Do we keep the marketing line ? If so, should we make this a partial that we can add to all guides so we can keep messaging consistent.